composer/helpers/v1/composer.json

```diff
@@ -5,7 +5,7 @@
         "composer/composer": "^1.10.9"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.9",
+        "friendsofphp/php-cs-fixer": "^3.0",
```

Comment:
So after chasing this a while, this won't work:
* `php-cs-fixer` requires `composer/semver` >= 3.0: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/abf6c314b99bd6dc156ab8e39967956141d8594c/composer.json#L20 - this bump landed as part of the `3.0` milestone for `php-cs-fixer`, so can't just downgrade to an earlier v3 release.
* `composer` v1 requires `composer/semver` ^1: https://github.com/composer/composer/blob/9c86ff9fff272bd0953c58ec793e62626dafaa00/composer.json#L27

Bummer, but not a huge deal given that composer v1 support is essentially EOL at this point, and this PR gave me what I wanted which was an opportunity to learn how Composer works.